package.json

```diff
@@ -1,30 +1,49 @@
 {
-  "name": "datastore.abstract",
-  "version": "0.0.0",
-  "description": "datastore",
-  "main": "index.js",
+  "name": "interface-datastore",
+  "version": "1.0.0",
+  "description": "datastore interface",
+  "main": "src/index.js",
   "scripts": {
-    "test": "node test.js"
+    "lint": "aegir-lint",
+    "build": "aegir-build",
+    "test": "aegir-test",
+    "flow": "flow",
+    "test:node": "aegir-test --env node",
+    "test:browser": "aegir-test --env browser",
+    "release": "aegir-release",
+    "release-minor": "aegir-release --type minor",
+    "release-major": "aegir-release --type major",
+    "coverage": "aegir-coverage",
+    "coverage-publish": "aegir-coverage publish"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipfs/interface-datastore.git"
   },
   "keywords": [
-    "datastore",
-    "abstract"
+    "interface",
+    "key-value",
+    "ipfs",
+    "datastore"
   ],
-  "author": "Juan Benet <juan@benet.ai> (http://juan.benet.ai/)",
+  "author": "Friedel Ziegelmayer <dignifiedquire@gmail.com>",
```

Comment:
Well, Juan kind of created the whole thing.

Mind adding yourself as contributor?